backend/level_slice.py

```diff
@@ -24,24 +24,21 @@
 class LevelSlice:
     """A class for processing slice and its records."""
 
-    def __init__(self, filename=None, filenames=None):
+    def __init__(self, filename, bucket=None):
         """Initialises slice object.
 
-        filname and filenames cannot be None at the same time.
         filename is used to load and save for single slice, and filenames is used to
         read multiple slices at the same time.
 
         Args:
-            filename: (optional) A string of the path to the slice.
-            filenames: (optional) A list of string that represent the path to slices.
+            filename: A string of the path to the slice.
+            bucket: An bucket object.
```

Comment:
Looks when the bucket is None, it means the raw csv file is on local host. 
Please remove the code to support local file.